### ISSUE TITLE
GitPad not using default text editor

### ISSUE THREAD

**Issue by AdamAndersonFalafelSoftware** [github/GitPad#32]
I am using the GitPad that comes bundled with GitHub for Windows. My initial text editor was set to the default Notepad, and commit messages opened for editing in that as expected. Then I changed my default program for .txt files to Sublime Text, and that caused commit messages to open in Sublime as expected. However, I quickly decided that I didn't like that behavior because of how the message would open in a tab but I'd have to close the entire editor to get the commit to proceed. So I changed the default program for .txt back to Notepad, but now GitPad seems "stuck" on using Sublime Text and I can't seem to convince it to unstick.

GitPad is the editor. At the Git Shell prompt, typing
`$env:EDITOR`
outputs
`GitPad`
I can prove that my default editor for text files is Notepad by opening Git Shell and typing
`.\test.txt`
This opens test.txt in Notepad. Then I try committing some changes with
`git commit`
and the commit message opens in Sublime Text.

Please help diagnosing.

**Comment by shiftkey** [comment]
Can you show which program is configured as the default for `.txt` files? That's the [extension](https://github.com/github/GitPad/blob/master/Program.cs#L60) we're using here when GitPad is invoked, I just want to confirm this is as it should be.

Here's what I've got on my Windows 10 VM:

<img width="443" src="https://cloud.githubusercontent.com/assets/359239/12691082/ab84dc3e-c738-11e5-8da7-0b0857fbc733.png">

**Comment by AdamAndersonFalafelSoftware** [comment]
Here's what appears in Control Panel | Programs | Set Associations:
![image](https://cloud.githubusercontent.com/assets/2729807/12692537/726ccda2-c6c6-11e5-8480-945388cea8d3.png)

...aaand I just figured it out. :) I went into Git Bash and typed
`git config core.editor`
and found that somehow unbeknownst to me, (but knownst to Git Shell) my core.editor setting had been set explicitly to ST. Git checks this setting before looking at environment variables. Removing the setting allowed the environment EDITOR setting to take over. Issue resolved; thanks for acting as my rubber duck!
